### ISSUE TITLE
Can't connect to Remote Preview

### ISSUE THREAD

**Issue by Bootfit** [Scirra/Construct-3-bugs#861]
In my project I selected Menu > Project > Remote Preview

Get error when remote preview tries to load/connect...

## Attach a .c3p

*A minimal Construct 3 project (.c3p) is required to be attached. Please see the guidelines*

## Steps to reproduce

1. Menu > Project > Remote Preview
2. Scan QR code in phone
3. Get error

## Observed result

Get popup with QR Code - scan code on phone and get the following error on my phone:
**Error: ReferenceError: Can't find variable: RemotePreviewPeerConnection**

And get the following message in the QR popup window:
1. **Unknown / unknown** (FFYWLU): Connecting… 
Waiting for runtime stats… 
0 bytes/s upload – 0 bytes total upload – Video: not yet available

## Expected result

*What did you expect to happen instead?*

## Affected browsers

* Chrome:
* Firefox: 
* Edge:
* Safari: yes - iPhone6S Plus

## System details

*If you see a crash report dialog, please copy and paste it to where it says "PASTE HERE" below. Otherwise please go to **Menu** -> **About** -> **Platform information** and paste that information there instead.*

<details><summary>View details</summary>

PASTE HERE

</details>

**Comment by Bootfit** [comment]
No .cP3 attached as the bug appears on any project, including the Scirra-built tutorials.

**Comment by shortercode** [comment]
What version of iOS are you running? WebRTC Data Channels was only just added in Safari 11, so remote preview will only work on iOS 11.

**Comment by Bootfit** [comment]
Yep - I'm running the latest iOS 11. Was working fine on Friday then on Saturday I started getting the error.

**Comment by Bootfit** [comment]
I also get this error when trying to remote preview on iPad running iOS 11...

![img_2548](https://user-images.githubusercontent.com/27493296/31074504-e40929e0-a768-11e7-964b-a38560d84e3a.png)

**Comment by AshleyScirra** [comment]
I can't reproduce the original problem on either an iPhone 5S or an iPad Air 2 running iOS 11. You didn't provide a project so I ran Space Blaster and it worked fine. The error looks like a script simply failed to load. Maybe there was a network error. You might want to clear your cache and try again.

The other error you posted is unrelated. I guessed from the error message that it was to do with using WebM Opus audio in projects and then remote previewing in Safari. It does not reproduce in projects that do not use WebM Opus. I filed #867 to track that.
